docs/admin/optionals.rst

```diff
@@ -264,9 +245,8 @@ Following operations are subject to rate limiting:
 | Adding to glossary                | ``GLOSSARY``       |               30 |               60 |            600 |
 +-----------------------------------+--------------------+------------------+------------------+----------------+
 
-Additionally if there are more than :setting:`AUTH_LOCK_ATTEMPTS` failed
-authentication attempts on one account, this account password authentication is
-disabled and it's not possible to login until user asks for password reset.
```

Comment:
@nijel answer appreciated

Reply:
The AUTH_LOCK_ATTEMPTS is there to avoid password guessing.

CAPTCHA doesn't prevent it completely (especially the simple one we have, see https://github.com/WeblateOrg/weblate/issues/1462).

Two-factor doesn't prevent guessing passwords at all - it is always second step after password is verified. So, even if it would not be possible to sign in to Weblate, it would be possible to determine user password and use it elsewhere (most people do reuse passwords).

And the password reset is rate limited as well.